setup.py

```diff
@@ -56,8 +56,7 @@
 
     entry_points={
         'console_scripts': [
-            'asr=asr.__main__:main',
```

Comment:
Please keep `asr` with a deprecation warning.

Reply:
You mean only the entry point, or the whole library (I.e. being able to ‘import asr’)?

Reply:
This remark is on the entry point. That's a must-have. The general remark is on the `import asr`.